### DISCUSSION TITLE
Banner Ads - Auto Refresh interval?

### DISCUSSION THREAD

**Discussion by DrOwlDev** [marcojak/MauiMTAdmob#63] [Q&A]
Hi Experts,

quick question with Banners, I can see after calling myAds.LoadAd(); 
it will reload and show a new impression after ~70s
>>>
[0:] 0__MyAds_AdLoaded
[0:] 0__MyAds_AdVImpression___GapBetweenLastImpression=70s
<<<

Do you know what's the minimum time a banner much be display before getting the impression credit? is the 70s configurable? and is it set to 70s for a reason? thanks.

**Comment by marcojak** [comment]
You can configure this from your Admob account and Google will automatically refresh the banner.
Or you can decide to avoid the automatic refresh and just load the new banner manually.
For the credit, it depends on the ads...many are pay per click, so the refresh interval doesn't matter
